common/pref_names.h

```diff
@@ -74,6 +74,10 @@ extern const char kBraveShieldsSettingsVersion[];
 extern const char kBinanceAccessToken[];
 extern const char kBinanceRefreshToken[];
 extern const char kDefaultBrowserPromptEnabled[];
+
+// Web discovery project
+extern const char kWebDiscoveryEnabled[];
```

Comment:
this should have an ENABLE_EXTENSIONS guard to match other usages

Reply:
Fixed in https://github.com/brave/brave-core/pull/9381/commits/f4e1718839df9bf2771744c054f4e7223a3207d0